Makefile

```diff
@@ -119,6 +119,12 @@ up/bootstrap: get vendor/hosts
 		docker-compose -f services/$${svc}/docker-compose.yml up -d 2>&1 | tee -a docker-compose.err; \
 	done
 	@source ./bin/helper.sh
+	@docker exec main_chain neo-go wallet nep17 transfer --force --wallet-config /wallets/config.yml -r http://main-chain.neofs.devenv:30333 --from NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP  --to NbUgTSFvPmsRxmGeWpuuGeJUoRoi6PErcM --token GAS --amount 1000
```

Comment:
how temporary is it?

Reply:
What temporary?

Reply:
i mean this looks like the most complicated target in our Makefiles. dont want to say it is bad but we tried to put such scripts in some `.sh` files before

Reply:
1. Things in `bin` are mostly reusable in some other contexts, deposit/tick/config are all relevant for manual execution, this one not so much (like `neofs-adm` below).
2. Those scripts are more generic and fancy, allowing overrides of various sorts, but here we have to use two different neo-go binaries and have to do very specific things.
3. `neofs-adm` invocations below are there, they're not scripted.
4. It's just four lines, really.

Reply:
can accept it